python/cudf/cudf/core/column/column.py

```diff
@@ -1414,6 +1414,31 @@ def as_column(arbitrary, nan_as_null=None, dtype=None, length=None):
     return data
 
 
+def _arr_from_array_interface_(array_interface=None, copy=False):
+    """Generates numpy array from memory address
+    https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.interface.html
+
+    Parameters
+    ----------
+    array_interface : dict
+        Contains `__array_interface__` which has pointer to
+        the memory address for array re-construction.
+
+    copy : bool
+        Copy array.  Default False
+
+    read_only_flag : bool
+        Read only array.  Default False.
+    """
+
+    class numpy_holder:
+        pass
+
+    holder = numpy_holder()
+    holder.__array_interface__ = array_interface
+    return np.array(holder, copy=copy)
```

Comment:
This seems very unnecessary. Couldn't we just use `np.asarray(...)` inplace of this entire function?